external/include/libcxx/locale

```diff
@@ -710,12 +710,12 @@ __num_get_signed_integral(const char* __a, const char* __a_end,
     if (__a != __a_end)
     {
         typename remove_reference<decltype(errno)>::type __save_errno = errno;
-        errno = 0;
+	set_errno(0);
```

Comment:
How about using tab or space?
Under 1tab=4space, It seems like below
```
        typename remove_reference<decltype(errno)>::type __save_errno = errno;
    set_errno(0);
```

Reply:
Done